server/pxf-api/src/main/java/org/greenplum/pxf/api/security/SecureLogin.java

```diff
@@ -65,6 +67,8 @@ public class SecureLogin {
     public static final String CONFIG_KEY_SERVICE_USER_IMPERSONATION = "pxf.service.user.impersonation";
     public static final String CONFIG_KEY_SERVICE_USER_NAME = "pxf.service.user.name";
 
+    public static final float DEFAULT_TICKET_RENEW_THRESHOLD = 0.8f;
```

Comment:
```suggestion
    public static final float DEFAULT_TICKET_RENEW_WINDOW = 0.8f;
```
see previous comments as to why